lib/src/strings.dart

```diff
@@ -235,3 +235,16 @@ const dartHandleUsr = 'c:@S@_Dart_Handle';
 
 const ffiNative = 'ffi-native';
 const ffiNativeAsset = 'asset';
+
+Directory? _tmpDir;
+
+/// A path to a unique temporary directory that should be used for files meant
+/// to be discarded after the current execution is finished.
+String get tmpDir {
+  if (Platform.environment.containsKey('TEST_TMPDIR')) {
+    return Platform.environment['TEST_TMPDIR']!;
```

Comment:
Maybe we should put something about `TEST_TMPDIR`being used for generating Macros in the readme?